build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: flow-aggregator
+  namespace: flow-aggregator
+spec:
+  selector:
+    app: flow-aggregator
+  type: NodePort
```

Comment:
maybe I am missing something, but why is this a `NodePort` Service? a ClusterIP is enough for the agent to connect to the aggregator

Reply:
Yes, I have removed the NodePort type declaration in the second PR, sorry for not merging it into this PR.